docker/Dockerfile.thumbv6m-none-eabi

```diff
@@ -10,14 +10,22 @@ RUN /cmake.sh
 COPY xargo.sh /
 RUN /xargo.sh
 
-COPY qemu.sh /
 RUN apt-get update && apt-get install --assume-yes --no-install-recommends \
     gcc-arm-none-eabi \
-    libnewlib-arm-none-eabi && \
-    /qemu.sh arm
+    libnewlib-arm-none-eabi \
+    libstdc++-arm-none-eabi-newlib
```

Comment:
These targets actually support C++, and `gcc-arm-none-eabi` actually contains the C++ compiler, just not `libstdc++v3`. If we install that, these targets support C++ just fine.